src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/approvedraftorders/ApproveDraftOrdersAboutToStartHandler.java

```diff
@@ -0,0 +1,175 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler.approvedraftorders;
```

Comment:
You could make the package structure consistent with what you have in model/ccd:
uk.gov.hmcts.reform.finrem.caseorchestration.handler.draftorders.judgeapproval;